lib/presentation/model/file/document_presentation_file.dart

```diff
@@ -58,7 +58,7 @@ class DocumentPresentationFile extends Equatable implements PresentationFile {
   final String sha256sum;
   final bool hasThumbnail;
   final int shared;
-  final MediaType mediaType;
+  final MediaType? mediaType;
```

Comment:
why you change it to nullable?

Reply:
When I switched to null-safety I forgot to delete it. It is none-nullable